superset/assets/stylesheets/less/variables.less

```diff
@@ -107,9 +107,21 @@
 /* Z-INDEX                                                              */
 /* Think of the site as "layers" rather than an arms race of numbers    */
 /* Try to keep these to a minimum                                       */
+/* Try to label semantic "layers" and add comments for usage notes      */
```

Comment:
Let's remove the `Try to` here

Reply:
![There is no try](https://media2.giphy.com/media/pvDp7Ewpzt0o8/giphy.gif)